common_test/src/main/java/uk/gov/dhsc/htbhf/eligibility/model/testhelper/CombinedIdAndEligibilityResponseTestDataFactory.java

```diff
@@ -22,77 +25,77 @@ public static CombinedIdentityAndEligibilityResponse anIdMatchFailedResponse() {
     public static CombinedIdentityAndEligibilityResponse anIdMatchedEligibilityNotConfirmedResponse() {
         return CombinedIdentityAndEligibilityResponse.builder()
                 .identityStatus(IdentityOutcome.MATCHED)
-                .eligibilityStatus(EligibilityOutcome.NOT_CONFIRMED)
-                .addressLine1Match(VerificationOutcome.NOT_SET)
-                .postcodeMatch(VerificationOutcome.NOT_SET)
-                .mobilePhoneMatch(VerificationOutcome.NOT_SET)
-                .emailAddressMatch(VerificationOutcome.NOT_SET)
```

Comment:
I wasn't keen on using static imports here because the enums for EligibilityOutcome and VerificationOutcome can be easily confused, so I think having the full qualification in this context helps.

Reply:
is it only for VerificationOutcome and EligibilityOutcome or for all static imports in this file?

Reply:
Yes, only those two as its those that might cause confusion, the others seem fine.